### PR TITLE
Remove usage of unrouted and untriaged labels

### PR DESCRIPTION
**Pull request by hubertdeng123**
👋

### CODE REVIEW

File: src/utils/businessHours.test.ts [hunk 1]
```diff
@@ -279,7 +279,9 @@ describe('businessHours tests', function () {
     });
 
     it('should calculate SLO violation if label is unrouted', async function () {
-      const result = await calculateSLOViolationRoute(UNROUTED_LABEL);
+      const result = await calculateSLOViolationRoute(
```
Comment (azaslavsky) [inline src/utils/businessHours.test.ts:282]: nit: Not strictly in this PR, but still confusing: on L262, I found it pretty hard to parse the double negative "not unrouted". Maybe just "if label is routed"?

**Review by azaslavsky — APPROVED** [review]

**Review by chadwhitacre — APPROVED** [review]
This PR seems to do a fine job of search/replacing `Status` for `Waiting for` labels. I think there's some deeper refactoring that could go along with this, but no need to block on that here.
